src/_variables.scss

```diff
@@ -0,0 +1,82 @@
+$colors: (
```

Comment:
Find and follow my previous comment about what variables should be extracted into this file.

**This comment is NOT related to colors.** You just ignored my prev comment, so I won't point the exact vars.